platform-includes/crons/setup/dotnet.mdx

```diff
@@ -18,3 +18,87 @@ If your job execution fails, you can notify Sentry about the failure:
 // 🔴 Notify Sentry your job has failed:
 SentrySdk.CaptureCheckIn("<monitor-slug>", CheckInStatus.Error, checkInId);
 ```
+
+## Heartbeat
+
+Heartbeat monitoring notifies Sentry of a job's status through one check-in. This setup will only notify you if your job didn't start when expected (missed). If you need to track a job to see if it exceeded its maximum runtime (failed), use check-ins instead.
+
+```csharp
+// Execute your scheduled task...
+
+// 🟢 Notify Sentry your job completed successfully:
+SentrySdk.CaptureCheckIn(
+    "<monitor-slug>",
+    CheckInStatus.Ok,
+    duration: TimeSpan.FromSeconds(10)
+);
+```
```

Comment:
Is the example code demonstrating heartbeat monitoring? I'm a little confused since the preceding paragraph says to use checkins to see if a job exceeds it's maximum runtime.. then the code calls the `CaptureCheckIn` API, but all of this is in a section titled `Heartbeat`.

Edit: Ah, I see... so in contrast to the regular checkin, this API call has a duration argument. Maybe worth stating that explicitly in the text above?

Reply:
The main difference between a heart-beat style check-in is that you do not send a `in_progress` check-in. Just `OK` or `Error`.

Let me make it more clear though!